src/components/il-tabs/il-tabs.component.js

```diff
@@ -25,42 +25,62 @@ class TabComponent extends LitElement {
     this.compact = this.offsetWidth < TabComponent.compactSizePixelWidth;
   }
 
-   getLinkElement() {
-     return document.getElementById(this.linkedby);
+   addPolyfillForContainerSupprt() {
```

Comment:
It's not really important that you're polyfilling something. The method name should describe what you're doing, e.g. addResizeListeners()